starboard/shared/win32/posix_emu/pthread.cc

```diff
@@ -15,6 +15,10 @@
 #include <errno.h>
 #include <pthread.h>
 
+#ifdef __cplusplus

```

Comment:
No need of the c++ check as this is a c++ file. 
The check is useful in the headers.